README.md

```diff
@@ -4,7 +4,41 @@ React hooks for fetching data from gtfs sources.
 
 ## Usage
 
-TODO: Describe.
+There are two hooks which this package provides. `useGtfsSchedule` and `useGtfsRealtime`. Both take two arguments, a resolver and a timeout.
+
+The resolver must be an async function which takes no arguments and returns a `Uint8Array` containing the raw data for the hook to process.
+For the schedule hook, this should be the raw zip data coming from your GTFS Schedule endpoint.
+For the realtime hook, this should be the protobuf binary data from your GFTS Realtime Alerts endpoint.
+
+**It is the callee's responsibility to ensure the resolver returns a valid `Uint8Array`.** The hook return `null` if this is not the case.
```

Comment:
```suggestion
**It is the callee's responsibility to ensure the resolver returns a valid `Uint8Array`.**
```
This also technically isn't true - if you return something other than `{Uint8Array|null|undefined}` an exception will likely get raised during parsing (the only try catch is around the resolver itself) and thus will go completely uncaught.